code/parachain/frame/composable-traits/src/assets.rs

```diff
@@ -25,7 +26,9 @@ impl BasicAssetMetadata {
 
 #[derive(Decode, Encode, Debug, Clone, PartialEq, Eq)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub struct Asset {
-	pub name: Vec<u8>,
+pub struct Asset<ForeignId> {
+	pub name: Option<Vec<u8>>,
 	pub id: u64,
```

Comment:
This should probably be a generic field instead of a concrete `u64`

Reply:
We should be ok with a `u128` no?

Reply:
Yes, but it would probably be best to make it generic. This can be addressed in another ticket though, feel free to resolve :+1:

Reply:
https://app.clickup.com/t/3mzpfw2